plugins/example-site/src/site/_site/index.html

```diff
@@ -0,0 +1,6 @@
+<html>
+  <head>
+    <title>Page title</title>
+  </head>
+  <body>Page body</body>
```

Comment:
can we add a simple integration test that just checks the content returned matches `Page body`?